WordPress/Sniffs/PHP/IniSetSniff.php

```diff
@@ -44,15 +44,16 @@ class IniSetSniff extends AbstractFunctionParameterSniff {
 	 * Array of PHP configuration options that are allowed to be manipulated.
 	 *
 	 * @since 2.1.0
+	 * @since 3.0.0 Renamed from `$whitelisted_options` to `$safe_options`.
 	 *
 	 * @var array Multidimensional array with parameter details.
-	 *     $whitelisted_options = array(
+	 *     $safe_options = array(
 	 *         (string) option name. = array(
 	 *             (string[]) 'valid_values' = array()
 	 *         )
 	 *     );
 	 */
-	protected $whitelisted_options = array(
+	protected $safe_options = array(
```

Comment:
Why `safe` here, instead of `allowed` (which matches the Summary)?

Reply:
Maybe I should update the summary ? The list is a minimal set of ini directives for which changing their value will not cause any problems for interoperability either between a plugin and WP or between plugins. In other words, these are not just _allowed_ to be changed, they are actually _safe_ to be changed.

Reply:
Addressed in: https://github.com/WordPress/WordPress-Coding-Standards/pull/2065/commits/be76478f0d283e6349e2cbde737c9d29e8d044e6